pkg/reader/ccip_interface.go

```diff
@@ -148,6 +148,13 @@ type CCIPReader interface {
 		destChainSelector cciptypes.ChainSelector,
 	) (rmntypes.RemoteConfig, error)
 
+	// GetRmnCurseInfo returns rmn curse/pausing information about the provided chains
+	// from the destination chain RMN remote contract. Caller should be able to access destination.
+	GetRmnCurseInfo(
+		ctx context.Context,
+		sourceChainSelectors []cciptypes.ChainSelector,
```

Comment:
```suggestion
		ctx context.Context,
		destChainSelector cciptypes.ChainSelector,
		sourceChainSelectors []cciptypes.ChainSelector,
```

Reply:
Dest chain selector should be in the OCR config, that can be fetched pretty easily in the plugin itself